framework/include/base/MooseError.h

```diff
@@ -181,13 +185,18 @@ mooseDeprecatedStream(S & oss, bool expired, Args &&... args)
   mooseDoOnce(std::ostringstream ss; mooseStreamAll(ss, args...);
               std::string msg = mooseMsgFmt(
                   ss.str(),
-                  "*** Warning, This code is deprecated and will be removed in future versions!\n",
+                  "*** Warning, This code is deprecated and will be removed in future versions:",
```

Comment:
Wait, are you sure you want to remove the newline here? Does this cause a pretty long line to be printed for most messages? I'm fine dropping the exclamation mark.

Reply:
The actual message generating function adds in it's own newlines - so I just removed some extras that made our messages a bit sparse.